getting_started.md

```diff
@@ -77,11 +77,13 @@ For any other future compile run you can simply do:
 ```
 sudo ./buildAll.sh
 ```
-If you have already set up the [FUN](#setting-up-the-fun), you can directly upload your binaries by calling
+If you have already set up the [FUN](#setting-up-the-fun) AND your compile machine is part of the FUN, you can directly upload your binaries to the FLUFFI ftp server (ftp.fluffi) by calling
```

Comment:
I would suggest lower case "and" here and small changes to fit in with the prose style of the rest of the getting started guide
```suggestion
If you have already set up the [FUN](#setting-up-the-fun) and it is reachable from the machine you use to compile, you can directly upload your binaries to the FLUFFI ftp server (ftp.fluffi) by calling
```